### DISCUSSION TITLE
Changes in Code doesn't show up

### DISCUSSION THREAD

**Discussion by Jelizog** [merkourisa/Augmented-Reality#12] [Q&A]
I'm making changes but those don't show up. 
The box stays the same color and either the sphere or the cylinder not showing.

That's my code:

`body style='margin: 0; overflow: hidden;'>

    <a-scene embedded arjs='sourceType: webcam;'>
        
      <a-marker preset="hiro">

        <a-sphere position="0 1.25 -5" radius="1.25" color="#EF2D5E">
          
        </a-sphere>
        
        <a-entity
        position="0 0 0"
        scale="0.05 0.05 0.05"
        gltf-model="your-server/https://raw.githack.com/AR-js-org/AR.js/master/aframe/examples/image-tracking/nft/trex/scene.gltf"
        ></a-entity>
      </a-marker>

      <a-entity camera></a-entity>
        
        <a-cylinder position="1 0.75 -3" radius="0.5" height="1.5" color="#FFC65D">

          scale="20 20 20";
        </a-cylinder>
        
        
        

        <a-box
            material="color: red;"
            scale="20 20 20"
            gps-entity-place="latitude: 44.506477; longitude: 11.301524;">
        </a-box>

        <a-camera
            rotation-reader
            gps-camera="simulateLatitude: 44.504348; simulateLongitude: 11.301027; minDistance: 1; maxDistance: 700;"
        >
		</a-camera>
	</a-scene>
</body>`

**Comment by merkourisa** [comment]
@Jelizog In the initial code you have the scene, inside the scene you have the **marker**, and inside the marker you the element **<a-entity id="box"** . You should import the cylinder and the sphere accordingly.
